web/src/state_data.ts

```diff
@@ -58,6 +58,7 @@ export const current_user_schema = z.object({
     is_owner: z.boolean(),
     onboarding_steps: z.array(onboarding_step_schema),
     user_id: z.number(),
+    has_zoom_token: z.boolean(),
```

Comment:
Please keep alphabetized lists alphabetized.

Also, the commit message should describe what your commit does: “Add has_zoom_token to current_user_schema”, not just “Update current_user_schema” (that could mean anything).